nominees/productive-forests.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Productive Forests",
+  "aliases": [
```

Comment:
If the alias is the same as the name, then it should not be included. Aliases should only be added when they add additional information not captured in the name

Reply:
Fixed in commit ca4bc51